content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -129,19 +159,26 @@ the :guilabel:`Reverse Transfer` pop-up window.
    :align: center
    :alt: View of a Helpdesk ticket with the return button highlighted.
 
-.. note::
+.. important::
    The :guilabel:`Return` button only appears on a ticket if the customer has a recorded delivery in
    the database.
 
-By default, the quantity will match the validated quantity from the delivery order. Update the
-:guilabel:`Quantity` field if necessary.
+Select a :guilabel:`Sales Order` or :guilabel:`Delivery to Return` to identify the products that
+need to be returned.
+
+By default, the quantity matches the validated quantity from the delivery order. Update the
+:guilabel:`Quantity` field if necessary. To remove a line, click the :guilabel:`🗑️ (trash can)`
+icon.
+
+Select a :guilabel:`Return Location` where the items should be directed after the return is
+completed.
 
 .. image:: after_sales/after-sales-reverse-transfer.png
    :align: center
    :alt: View of a reverse transfer creation page.
 
 Click :guilabel:`Return` to confirm the return. This generates a new warehouse operation for the
-incoming returned product(s). A :guilabel:`Return` smart button will then be added to the top of the
+incoming returned products. A :guilabel:`Return` smart button is then added to the top of the
 ticket.
```

Comment:
After returning, I am seeing the WH/IN/00033 (a return order) and not the ticket. Might want to mention to get back to the ticket with the breadcrumb menu or just to go back to the ticket).